### DISCUSSION TITLE
Copilot被告

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#491] [General]
大家覺得怎樣呢? Copilot沒有經過同意使用了大家在GitHub上的開源程式碼訓練AI是否屬於合理使用，以及事情之後會怎樣發展?
https://www.ithome.com.tw/news/154076

**Comment by xCk27x** [comment]
我覺得Github上的開源作品能被用來訓練AI
但如果是商業用途就不行了

**Comment by williamyen0623** [comment]
用過copilot都知道 這個技術的強大和方便
雖說是用心開發的產品 收費應該理所當然
但歸根究底 資源的來源是使用者（programmer）免費提供的，不免也有著作權的疑慮

**Comment by limitx0** [comment]
拿沒取得授權的程式碼來訓練真的不好
有些開源也不是可以隨意拿來運用

**Comment by skyling0299** [comment]
這聽起來跟畫圖的AI也有差不多的感覺耶，畫圖的那個AI也有被告侵犯著作權的疑慮
但說實話，寫程式的各位真的沒有過ctrl c ctrl v嗎...
